tests/Unit/Core/Domain/SpecificPrice/ValueObject/PriorityListTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Core\Domain\SpecificPrice\ValueObject;
+
+use Generator;
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Domain\SpecificPrice\Exception\SpecificPriceConstraintException;
+use PrestaShop\PrestaShop\Core\Domain\SpecificPrice\ValueObject\PriorityList;
+
+class PriorityListTest extends TestCase
+{
+    /**
+     * @dataProvider getValidPriorities
+     *
+     * @param string[] $priorities
+     */
+    public function test_creates_object_with_valid_priorities(array $priorities): void
```

Comment:
please use camel case for test function names
```suggestion
    public function testCreatesObjectWithValidPriorities(array $priorities): void
```

Reply:
I was sure prestashop started using snake case as its getting accepted as a better practice for tests (increases readability).
Buut apparently I've confused it with some other project. (sorry :face_with_head_bandage: )